README.md

```diff
@@ -327,12 +330,17 @@ The following is the progress report:
 
 ## More information
 
-### Further reading `IAsyncEnumerable`
+### The AsyncSeq library
+
+If you're looking to use `IAsyncEnumerable` with `async` and not `task`, the existing [`AsyncSeq`][11] library already provides excellent coverage of that use case. While `TaskSeq` is intended to interoperate with `async` as `task` does, it's not intended to provide an `AsyncSeq` type (at least not yet).
```

Comment:
```suggestion
The [`AsyncSeq`][11] and `TaskSeq` library both operate on asynchronous sequences, but there are a few fundamental differences, most notably that the former _does not_ implement `IAsyncEnumerable<'T>`, but has its own same-named, but differently behaving type. 

There are more differences:

|                        | `TaskSeq`                                                   | `AsyncSeq`                                                           |
|------------------------|-------------------------------------------------------------|----------------------------------------------------------------------|
| Frameworks             | .NET 5.0+, NetStandard 2.1                                  | .NET 5.0+, NetStandard 2.0 and 2.1, .NET Framework 4.6.1+            |
| Underlying type        | `System.Collections.Generic.IAsyncEnumerable<'T>            | Its own type, also called `IAsyncEnumerable<'T>`, but not compatible |
| Implementation         | State machine (statically compiled)                         | No state machine, continuation style                                 |
| Semantics              | `seq`-like: on-demand                                       | `seq`-like: on-demand                                                |
| Support `let!`         | All task-like (async, task, valuetask, Task, or GetAwaiter) | `async` only                                                         |
| Support `yield!`       | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence            | `AsyncSeq`                                                           |
| Support `for`          | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence            | `AsyncSeq` any sequence                                              |
| Conversion to other    | `TaskSeq.toAsyncSeq`                                        | `AsyncSeq.toAsyncEnum`                                               |
| Conversion from other  | Implicit (yield!) or `TaskSeq.ofAsyncSeq`                   | `AsyncSeq.ofAsyncEnum`                                               |
| Recursion in `yield!`  | No (requires F# support, upcoming)                          | Yes                                                                  |
| Based on F# concept of | `task`                                                      | `async`                                                              |
| MoveNext operation     | `ValueTask`                                                 | Cold-started `Task`                                                  |
| Cancellation           | Implicit token governing iteration                          | Implicit token passed to each subtask                                |
| Performance            | Very high, negligible allocations                           | Slower, more allocations, due to using `async`                       |
| Behavior with `Return` | No allocations, no `Task` created                           | Allocated, a `Task` will be created                                  |
| Parallelism            | Possible with ChildTask, support will follow                | Supported explicitly                                                 |
|                        |                                                             |                                                                      |
```

Reply:
I'll put this one in a separate PR, but I think it is useful.

Reply:
See #131

Reply:
This looks wonderful ❤️